dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/utils/DataMetaInformationSearchFilter.kt

```diff
@@ -1,20 +1,50 @@
 package org.dataland.datalandbackend.repositories.utils
 
 import com.fasterxml.jackson.annotation.JsonIgnore
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Schema
 import org.dataland.datalandbackend.model.DataType
 import org.dataland.datalandbackendutils.model.QaStatus
+import org.dataland.datalandbackendutils.utils.swaggerdocumentation.BackendOpenApiDescriptionsAndExamples
+import org.dataland.datalandbackendutils.utils.swaggerdocumentation.GeneralOpenApiDescriptionsAndExamples
 import java.util.UUID
 
 /**
  * A filter class used in the searchDataMetaInformation()-Method which allows
  * convenient usage of SEPL instructions in the query
  */
 data class DataMetaInformationSearchFilter(
+    @field:Schema(
+        description = GeneralOpenApiDescriptionsAndExamples.COMPANY_ID_DESCRIPTION,
+        example = GeneralOpenApiDescriptionsAndExamples.COMPANY_ID_EXAMPLE,
+    )
     val companyId: String? = null,
+    @field:Schema(
+        description = GeneralOpenApiDescriptionsAndExamples.DATA_TYPE_DESCRIPTION,
+    )
     val dataType: DataType? = null,
+    @field:Schema(
+        description = GeneralOpenApiDescriptionsAndExamples.REPORTING_PERIOD_DESCRIPTION,
+        example = GeneralOpenApiDescriptionsAndExamples.REPORTING_PERIOD_EXAMPLE,
+    )
     val reportingPeriod: String? = null,
+    @field:Schema(
+        description = BackendOpenApiDescriptionsAndExamples.IS_ONLY_ACTIVE_DESCRIPTION,
```

Comment:
I don't think, this is correct. The flag "only active" is a "show only active" but the description here is "Boolean flag indicating whether the dataset is currently the only active dataset". I think this should be 
```suggestion
        description = BackendOpenApiDescriptionsAndExamples.SHOW_ONLY_ACTIVE_DESCRIPTION,
```